### ISSUE TITLE
Battle Ready Trait messes with Focus value

### ISSUE THREAD

**Issue by PascalMueller1988** [mjording/marvel-multiverse#68]
When I try to reduce the focus of a character with the Battle Ready trait the current focus value always jumps up by 30.

Example:
Max focus is 150. Character spends 60 focus, so I update the focus to 90. When closing the character sheet it returns to 120.

**Comment by mjording** [comment]
having trouble duplicating this. Are you updating focus via token or sheet?

**Comment by PascalMueller1988** [comment]
For me it happened with both. Let me see if it's still happening.

**Comment by PascalMueller1988** [comment]
Yes, it's still happening and happens through both sheet and token.

To explain the behavior again:
If I change the focus on a character token that has the Battle Ready trait, the value I enter gets modified by +30.

Example 1: If the max focus would be 210 and I enter -60 I end up with 180.
Example 2: If I set the value at 120 it ends up at 150.
Example 3: If I set the value to 200 (so less than 30 below max) it ends up at 210.
